distributions/aws/examples/cognito/README.md

```diff
@@ -33,7 +33,7 @@ This guide assumes that you have:
         ```
 1. AWS IAM permissions to create roles and attach policies to roles.
 
-1. Clone the `awslabs/kubeflow-manifest` repo.
+1. Clone the `awslabs/kubeflow-manifest` repo and checkout release branch.
```

Comment:
nit: Calling it out as release branch here as of right now it doesn't currently exist? Also how will we handle the different versions i.e (1.3 ... 1.4)? maybe state desired release branch?

Reply:
the branch we are working on are release branches, e.g. v1.3-branch but this will be changed when we get closer to release since we will cut a tag